lib/doorkeeper/config.rb

```diff
@@ -321,6 +321,9 @@ def configure_secrets_for(type, using:, fallback:)
     #
     option :realm,                          default: "Doorkeeper"
 
+    # Issuer URL for the OAuth Authorization Server
+    option :issuer,                         default: nil
```

Comment:
Technically this is maybe OpenID more, but OAuth 2.0 does have a spec for it, see: #1720